egs/aspire/s5/local/multi_condition/aspire_data_prep.sh

```diff
@@ -22,12 +21,11 @@ tmpdir=`pwd`/data/local/data
 mkdir -p $tmpdir
 if [ $stage -le 0 ]; then
 
-  find $dev_transcript/ -name '*.stm'  > $tmpdir/transcripts.flist
+  find $dev_transcript/ -name 'dev.stm'  > $tmpdir/transcripts.flist
   find $dev_audio/ -name '*.wav'  > $tmpdir/wav.flist
   find $test_audio/ -name '*.wav'  > $tmpdir/wav_test.flist
-  find $eval_audio/ -name '*.wav'  > $tmpdir/wav_eval.flist
 
-  n=`cat $tmpdir/transcripts.flist | wc -l`
+  n=`cut -d' ' -f1 $(cat $tmpdir/transcripts.flist) | uniq | wc -l`
```

Comment:
better use awk. Relying cut might cause issues if the field separator was changed to other whitespaces e.g. tabs.